misc/python/materialize/output_consistency/ignore_filter/inconsistency_ignore_filter.py

```diff
@@ -231,7 +225,6 @@ def is_function_taking_shortcut(expression: Expression) -> bool:
 
         for expression in expressions:
             if expression.contains(is_function_taking_shortcut, True):
-                # see https://github.com/MaterializeInc/materialize/issues/17189
                 return True
```

Comment:
Could we keep a comment here like `TODO: Remove when #17189 is fixed`?

Reply:
It is still there as a `YesIgnore` where the return value is consumed. Therefore, I don't think it is necessary.